pkg/background/update_request_controller.go

```diff
@@ -370,6 +370,50 @@ func (c *controller) deletePolicy(obj interface{}) {
 	}
 }
 
+func (c *controller) deleteNSPolicy(obj interface{}) {
```

Comment:
Most of the logic applied to both clusterpolicy and policy, can ewe extract and re-use it if possible?

Reply:
@sandeshlmore ?